packages/table/src/headers/columnHeader.tsx

```diff
@@ -103,18 +103,13 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
     }
 
     private wrapCells = (cells: Array<React.ReactElement<any>>) => {
-        const { grid, viewportRect, columnIndexStart } = this.props;
+        const { grid } = this.props;
```

Comment:
`ColumnHeader` simply scrolls with the regular body cells now; it doesn't need to do fancy `transform`s anymore.